### PR TITLE
feat(node-analyzer): bumped RuntimeScanner and HostScanner to latest versions

### PR DESCRIPTION
**Pull request by ProvoK**
Extended Changelog:
* HostScanner, bumped to 0.3.6: 
  * Security fix for CVE-2022-41721 
  * Fix AcceptRisk evaluation wrong in some edge cases
* RuntimeScanner, bumped to 1.4.6: 
  * Fix AcceptRisk evaluation wrong in some edge cases

NOTE: IIRC changelog changes are not needed if we provide the correct commit body payload. Please lmk if it's not the case @sysdiglabs/team-tools-agent

### CODE REVIEW

File: charts/sysdig-deploy/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sysdig-deploy
 description: A chart with various Sysdig components for Kubernetes
 type: application
-version: 1.5.57
```
Comment (aroberts87) [inline charts/sysdig-deploy/Chart.yaml]: Hey @ProvoK ! You no longer need to manually increment the  `sysdig-deploy` chart version or the dependency versions, we have a GitHub action that does that automatically now if a dependent subchart is modified (https://github.com/sysdiglabs/charts/blob/master/.github/workflows/update-sysdig-deploy-chart.yaml). Can you please remove the edits to `charts/sysdig-deploy/Chart.yaml`?

**Review by aroberts87 — CHANGES_REQUESTED** [review]

File: charts/sysdig-deploy/Chart.yaml (hunk 1 above)
Comment (ProvoK) [reply charts/sysdig-deploy/Chart.yaml]: sure! very happy to do that 😄

Comment (ProvoK) [reply charts/sysdig-deploy/Chart.yaml]: just FYI: `node-analyzer` was at version `x.x.28` and `sysdig-deploy` was referencing `x.x.27`, so maybe something it's not working as intended, you may want to double check.

Comment (ProvoK) [reply charts/sysdig-deploy/Chart.yaml]: rollbacked changes with [8b314fc](https://github.com/sysdiglabs/charts/pull/916/commits/8b314fc5409e92426d13848d92131842d49613ee)

Comment (aroberts87) [reply charts/sysdig-deploy/Chart.yaml]: > just FYI: `node-analyzer` was at version `x.x.28` and `sysdig-deploy` was referencing `x.x.27`, so maybe something it's not working as intended, you may want to double check.

I'm going to look into this now. This shouldn't happen. 🤔

**Review by aroberts87 — APPROVED** [review]

File: charts/sysdig-deploy/Chart.yaml (hunk 1 above)
Comment (aroberts87) [reply charts/sysdig-deploy/Chart.yaml]: @ProvoK - `sysdig-deploy` is referencing `x.x.28` now (https://github.com/sysdiglabs/charts/blob/master/charts/sysdig-deploy/Chart.yaml#L34), but I think I now understand where the confusion is coming from. The `sysdig-deploy` chart is not bumped as a part of the merge containing the subchart changes, but rather a new PR is opened automatically with the version bumps. In this case that autogenerated PR is here: https://github.com/sysdiglabs/charts/pull/914. Those autogenerated PRs are approved by someone on @sysdiglabs/team-tools-agent. The version of `master` you were looking at may have been from a time between when the `node-analyzer` chart was bumped to `x.x.28` (https://github.com/sysdiglabs/charts/pull/907) and when I approved and merged the `sysdig-deploy` version bumps. Does that make sense?

**Pull request merged**